### ISSUE TITLE
Latex plot titles are not rendered in VS Code

### ISSUE THREAD

**Issue by MalteMederacke** [plotly/plotly.py#3661]
Hi,

I try to rename my plot axis using latex titles, they are not rendered in my VS Code jupyter notebook, nor in an exported html using include_mathjax = 'cdn'. 
```
fig = go.Figure(
    data=go.Scatter3d(x = [1,2,3],y=[2,3,1],z=[1,3,3]))
fig.update_layout(scene = dict(xaxis_title = r"$\lambda$"))
fig.show()
```
gives me:

![image](https://user-images.githubusercontent.com/65277828/161286021-7a173e75-b183-44c5-8955-ec296f6ab27b.png)

The same expression in a markdown cell works:

$\lambda$

![image](https://user-images.githubusercontent.com/65277828/161286205-9c31ee6f-44ff-44f3-87c2-9abbba41be5d.png)

Using maplotlib works, as well. 

I tried a workaround posted in #1740  using:
```
from IPython.display import display,HTML
display(HTML("""
<script type="text/javascript" async
  src="https://cdnjs.cloudflare.com/ajax/libs/mathjax/2.7.7/MathJax.js?config=config=TeX-AMS-MML_SVG">
</script>
"""))
```
 without success.

Any help is appreciated!

**Comment by IliasAarab** [comment]
I'm having the same issue, with the latest versions of Plotly and vscode stable

**Comment by Miladiouss** [comment]
I want to confirm that this doesn't work for 2-D plots either. However, when writing the figure as a PNG, it renders correctly.

**Comment by theo-brown** [comment]
[People have had the same issue due to running on different browsers](https://stackoverflow.com/questions/66713595/latex-in-y-axis-title-in-plotly-not-working), so it sounds like it's a rendering issue to do with how VS Code displays notebooks.

Given that VS Code is probably the most popular IDE, it would be nice if plotly did something to support it.

Note that this is also an issue in **Google Chrome** and **Firefox**, which are also quite popular browsers.

**Comment by JhianSV** [comment]
I confirm the problem persist, I agree with @herbzhao

**Comment by borondics** [comment]
I have the same problem.

Visual Studio Code
Version: 1.69.2 (Universal)
Commit: 3b889b090b5ad5793f524b5d1d39fda662b96a2a
Date: 2022-07-18T16:12:57.074Z (3 wks ago)
Electron: 18.3.5
Chromium: 100.0.4896.160
Node.js: 16.13.2
V8: 10.0.139.17-electron.0
OS: Darwin x64 21.5.0

Plotly version: 4.14.3

**Comment by ez2rok** [comment]
I'm having the same issue as well. My specs are:

Version: 1.72.2 (Universal)
Commit: d045a5eda657f4d7b676dedbfa7aab8207f8a075
Date: 2022-10-12T22:16:30.254Z
Electron: 19.0.17
Chromium: 102.0.5005.167
Node.js: 16.14.2
V8: 10.2.154.15-electron.0
OS: Darwin arm64 21.4.0
Sandboxed: No

Plotly version: 5.10.0

**Comment by Philipwur** [comment]
Also having this problem on VSCode, LATEX does render fine on Jupyter notebook (firefox) though.

**Comment by marthacryan** [comment]
Can anyone confirm whether this is still an issue?

**Comment by rbnhln** [comment]
> Can anyone confirm whether this is still an issue?

Yes, it is. I'm using plotly version 6.1.2 and the latest vscode version. My python version is 3.12.3.  

This workaround works for me:
```
import plotly
import plotly.graph_objects as go
from IPython.display import display, HTML
## Tomas Mazak's workaround
plotly.offline.init_notebook_mode()
display(HTML(
    '<script type="text/javascript" async src="https://cdnjs.cloudflare.com/ajax/libs/mathjax/2.7.1/MathJax.js?config=TeX-MML-AM_SVG"></script>'
))
```
